watch-library/hardware/main.c

```diff
@@ -69,7 +114,11 @@ int main(void) {
     // if date/time register is 0 (power on reset state), default year to 2023.
     watch_date_time date_time = watch_rtc_get_date_time();
     if (date_time.reg == 0) {
-        date_time.unit.year = 3;
+        date_time.unit.year = get_initial_year();
+        date_time.unit.month = get_initial_month();
+        date_time.unit.day = get_inital_day();
+        date_time.unit.hour = get_inital_hour();
+        date_time.unit.minute = get_inital_minute();
```

Comment:
Not sure if this is worth much, but it's been helpful for me. For Linux users who only add these 5 lines and don't change any other files, `sed` can do all the work (though it's a kind of hacky).

From `movement/make`:
```
sed -Ei "s/(unit.year = ).*(;)/\1$(( $(date +%-g) - 20 ))\2/;s/(unit.month = ).*(;)/\1$(date +%-m)\2/;s/(unit.day = ).*(;)/\1$(date +%-d)\2/;s/(unit.hour = ).*(;)/\1$(date +%-H)\2/;s/(unit.minute = ).*(;)/\1$(date +%-M)\2/" ../../watch-library/hardware/main.c && make COLOR=YOURCOLOR && make install COLOR=YOURCOLOR
```
I know it's not pretty, but it is handy.